tools/bazar/services/BazarListService.php

```diff
@@ -0,0 +1,244 @@
+<?php
+
+namespace YesWiki\Bazar\Service;
+
+use YesWiki\Bazar\Controller\EntryController;
+use YesWiki\Wiki;
+use YesWiki\Bazar\Field\BazarField;
+use YesWiki\Bazar\Service\EntryManager;
+use YesWiki\Bazar\Service\ExternalBazarService;
+use YesWiki\Bazar\Service\FormManager;
+
+class BazarListService
+{  
+    public function __construct(Wiki $wiki, EntryManager $entryManager, EntryController $entryController,
+                                ExternalBazarService $externalBazarService, FormManager $formManager)
+    {
+      $this->wiki = $wiki;
+      $this->entryManager = $entryManager;
+      $this->entryController = $entryController;
+      $this->externalBazarService = $externalBazarService;
+      $this->formManager = $formManager;
+    }
+
+    public function getForms($options) : array
+    {
+        // External mode activated ?
+        if ($options['externalModeActivated'] === true) {
+            return $this->externalBazarService
+                        ->getFormsForBazarListe($options['externalIds'], $options['refresh']);
+        } else {
+            return $this->formManager->getAll();
+        }
+    }
+
+    public function getEntries($options, $forms = null) : array
```

Comment:
```suggestion
    /**
     * @return array [$entries,$filters]
     */
    public function getEntriesAndFilters(array $arguments, $forms = null) : array
```
Je serai partisan de n'avoir que cette méthode publique qui rendrait les résultats de entries et filters en une seul fois car leur usage est toujours combiné

et je préfère conserver le terme `$arguments` que de changer à `$options`